cpu-profiling/cl-position-handler/README.md

```diff
@@ -0,0 +1,357 @@
+  ```
+  clinic bubbleprof --visualize-only .clinic/<PID>.clinic-bubbleprof
+  ```
+
+### Observations
+#### Profile: 892321.clinic-bubbleprof.html
+- Observed `23,659 ms in asynchronous delays, 19,145 ms in synchronous delays.` in mysql connect calls
+  ```
+  at Socket.Socket.connect node:net:993:7
+  at Connection.connect ../../node_modules/mysql/lib/Connection.js:76:13
+  at  ../../node_modules/@mojaloop/central-services-database/node_modules/knex/lib/dialects/mysql/index.js:56:18
+  at Client_MySQL.acquireRawConnection ../../node_modules/@mojaloop/central-services-database/node_modules/knex/lib/dialects/mysql/index.js:51:12
+  at create ../../node_modules/@mojaloop/central-services-database/node_modules/knex/lib/client.js:237:39
+  ```
+  By looking at the query log in mysql, the connections are being reused, so it seems like there is no issue.
+
+---
+
+## Mysql queries
+
+The following are the queries executed during a prepare / commit message handling operation.
+
+### Prepare
+```
+2023-08-21T12:04:15.916493Z        28 Query     select * from `settlementModel` where `isActive` = 1
+2023-08-21T12:04:15.918116Z        28 Query     select * from `participant`
+2023-08-21T12:04:15.918940Z        28 Query     select * from `participantCurrency` order by `participantCurrencyId` asc
+2023-08-21T12:04:15.920707Z        28 Query     BEGIN
+2023-08-21T12:04:15.921503Z        28 Query     select * from `transferStateChange` where `transferId` = '9d74acdb-d7da-4362-8f45-661d4fb2c12e' order by `transferStateChangeId` desc limit 1
+2023-08-21T12:04:15.922233Z        28 Query     select * from `participantPosition` where `participantCurrencyId` in (7, 8) for update
+2023-08-21T12:04:15.922971Z        28 Query     update `participantPosition` set `participantPositionId` = 7, `participantCurrencyId` = 7, `value` = 0, `reservedValue` = '2.0000', `changedDate` = '2023-08-21 12:04:15.921' where `participantPositionId` = 7
+2023-08-21T12:04:15.923564Z        41 Query     select * from `participantLimit`
+2023-08-21T12:04:15.924732Z        28 Query     update `participantPosition` set `value` = '2.0000', `reservedValue` = '0.0000', `changedDate` = '2023-08-21 12:04:15.921' where `participantPositionId` = 7
+2023-08-21T12:04:15.925035Z        28 Query     select * from `transfer` where `transferId` in ('9d74acdb-d7da-4362-8f45-661d4fb2c12e') for update
+2023-08-21T12:04:15.926852Z        28 Query     insert into `transferStateChange` (`createdDate`, `reason`, `transferId`, `transferStateChangeId`, `transferStateId`) values ('2023-08-17 15:25:07.000', NULL, '9d74acdb-d7da-4362-8f45-661d4fb2c12e', NULL, 'RESERVED')
+2023-08-21T12:04:15.928243Z        28 Query     insert into `participantPositionChange` (`participantPositionId`, `reservedValue`, `transferStateChangeId`, `value`) values (7, '0.0000', 5073, '2.0000')
+2023-08-21T12:04:15.928811Z        28 Query     COMMIT
+```
+
+### Commit
```

Comment:
```suggestion
### Commit (i.e. Fulfil)
```

Reply:
if You can capture the "smoothness" of the flow-mode vs recursive.

Reply:
Done